data/mods/ssb/abilities.ts

```diff
@@ -727,18 +729,51 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
 		shortDesc: "Changes the pokemon's form upon switch-in depending on the amount of pokemon still alive on the user's team.",
 		name: "The Numbers Game",
 		onStart(pokemon) {
```

Comment:
I would rename this variable to `target` to match the variable naming convention for other abilities, and then change `poke` to `pokemon` in `changeSet`

Reply:
OK